tests/src/test/scala/cats/tests/KleisliTests.scala

```diff
@@ -15,8 +15,25 @@ class KleisliTests extends CatsSuite {
   implicit def kleisliEq[F[_], A, B](implicit A: Arbitrary[A], FB: Eq[F[B]]): Eq[Kleisli[F, A, B]] =
     Eq.by[Kleisli[F, A, B], A => F[B]](_.run)
 
+  implicit val xorT = XorT.xorTEq[Kleisli[Option, Int, ?], Unit, Int]
+
+  implicit val optionApplicativeError : ApplicativeError[Option, Unit] = new ApplicativeError[Option, Unit] {
```

Comment:
Also curious how @ceedubs thinks about this, he suggested it "weird" in gitter right?